packages/nx/src/native/tests/workspace_files.spec.ts

```diff
@@ -243,7 +216,7 @@ describe('workspace files', () => {
       expect(getWorkspaceFilesNative(fs.tempDir, globs).projectFileMap)
         .toMatchInlineSnapshot(`
         {
-          "project1": [
+          "libs/project1": [
```

Comment:
We probably dont need this whole `describe('errors')` block anymore. We're not doing any parsing now.